device-types/Server Technology/C1W08HC-0ABA2BAC.yaml

```diff
@@ -0,0 +1,67 @@
+---
+manufacturer: Server Technology
+model: C1W08HC-0ABA2BAC
+slug: server-technology-c1w08hc-0aba2bac
+part_number: C1W08HC-0ABA2BAC
+u_height: 1
+is_full_depth: false
+airflow: passive
+comments: |
+  PRO1 Switched, Per-Inlet Power Sensing, Primary, 16A, 120-240V, (8) C13,
+  Dimensions: 45 x 437 x 178 mm (1.75 x 17.2 x 7.0")
+console-ports:
+  - name: ser
+    label: SER
+    type: rj-45
+  - name: th1
+    label: T/H1
+    type: rj-11
+  - name: th2
+    label: T/H2
+    type: rj-11
```

Comment:
A `console-port` is defined as:

> A console port provides connectivity to the physical console of a device. These are typically used for temporary access by someone who is physically near the device, or for remote out-of-band access provided via a networked console server.

Do these actually fit that description? What are these ports actually used for?

Reply:
These are temperature and humidity sensor ports, which match a previously accepted commit - see `device-types/Server Technology/C2WG24SN-YCLN5D6.yaml`

Reply:
If you'd like, I can remove these sensor ports from both my PR and the prior ServerTech units?